bedrock/stories/templates/stories/articles/art-of-engagement.html

```diff
@@ -0,0 +1,148 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "stories/stories-base-article.html" %}
+
+{% from "stories/includes/macros.html" import stories_article_blockquote, stories_image with context %}
+
+
+{% block publish_date %}2022-08-02{% endblock %}
+{% block publish_date_formatted %}August, 2, 2022{% endblock %}
```

Comment:
nit: No comma after the month.
```suggestion
{% block publish_date_formatted %}August 2, 2022{% endblock %}
```